packages/cozy-client/src/authentication/mobile.js

```diff
@@ -50,7 +58,14 @@ const authenticateWithSafari = url => {
     }
   })
 }
-
+/**
+ * Open an inAppBrowsr
+ * and resolve with the URL containing
+ * the token
```

Comment:
Opens
InAppBrowser
Resolves

I find it a bit less
readable when
sentence are split very often
by line breaks ;)